lstchain/reco/reconstructor.py

```diff
@@ -0,0 +1,951 @@
+import inspect
+import logging
+
+from iminuit import Minuit
+import numpy as np
+import matplotlib.pyplot as plt
+from copy import copy
+import astropy.units as u
+
+from lstchain.data.normalised_pulse_template import NormalizedPulseTemplate
+
+from ctapipe.core import Component
+from ctapipe.core.traits import Bool, Float, Int, Path
+from lstchain.calib.camera.pixel_threshold_estimation import get_bias_and_std
+from lstchain.io.lstcontainers import DL1LikelihoodParametersContainer
+from lstchain.visualization.camera import display_array_camera
+
+logger = logging.getLogger(__name__)
+
+try:
+    from lstchain.reco.log_pdf_CC import log_pdf_ll as log_pdf_ll
+    from lstchain.reco.log_pdf_CC import log_pdf_hl as log_pdf_hl
+    from lstchain.reco.log_pdf_CC import asygaussian2d as asygaussian2d
+except ImportError:
+    logger.warning('WARNING: Compiled likelihood reconstruction numbaCC functions missing.'
+                   ' If you plan to use the likelihood reconstruction you should run:\n'
+                   'lstchain/scripts/numba_compil_lhfit.py')
```

Comment:
I tried to integrate the compilation following the page you show but didn't manage to make it work. The issue was that this was not applicable to wheel based installation from what I could gather. A possible solution may be to have my code outside of lstchain and import it as a separated module (I guess removing current dependencies to lstchain, but this should be easy from what I see). If you have a solution It would be nice.